frontend/src/pages/meditation/components/meditation-list/meditation-list.tsx

```diff
@@ -2,7 +2,7 @@ import { BackButton } from '#libs/components/components.js';
 import { getValidClassNames } from '#libs/helpers/helpers.js';
 import { useCallback } from '#libs/hooks/hooks.js';
 
-import { mockedEntries } from '../libs/constants/constants.js';
+import { mockedEntries } from '../../libs/constants/constants.js';
```

Comment:
we no longer need mocked entries, let's remove them
```suggestion
```

Reply:
Done